openlibrary/templates/site/body.html

```diff
@@ -29,12 +29,13 @@
       $for flash in get_flash_messages():
         <div class="$flash.type"><span>$flash.message</span></div>
       </div>
-      $# Paste next line into announcement variable to show the blue banner
-      $# <strong>Title:</strong> Description &nbsp; <a href="#" data-ol-link-track="AnnouncementsBar|BlogWidget" class="btn primary">Learn More</a>
-      $ announcement = None
-      $if announcement:
-        <div class="page-banner page-banner-body">
-         $:announcement
-        </div>
+      $# To create a new announcement banner, please input the message within the announcement variable and
+      $# add a key with the following schema YYYY_MM_DD_Title_Of_Banner within the storage_key variable
+      $# If you are using the blue banner (see 'site/banner'), do add a title in the title variable
+      $ announcement = ''
+      $ storage_key = ''
+      $ title = ''
+      $ announcement_bar = render_template('site/banner', announcement, storage_key, title) if announcement and storage_key else ''
```

Comment:
```suggestion
      $ announcement_bar = render_template('site/banner', announcement, storage_key) if announcement and storage_key else ''
```
Remove unused parameter.